tlddoc/pom.xml

```diff
@@ -38,6 +37,14 @@ Tag Library Documentation Generator is a utility for automatically generating ja
                     <skip>true</skip>
                 </configuration>
             </plugin>
+            <plugin>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
+                <configuration>
+                    <release>11</release>
```

Comment:
With this set here, perhaps `maven.compiler.source/target` properties could be dropped?

Reply:
Ah, you're right. Thank you, let me update